x/perpetual/migrations/v10_migration.go

```diff
@@ -6,6 +6,14 @@ import (
 )
 
 func (m Migrator) V10Migration(ctx sdk.Context) error {
+	// Update params
+	params := m.keeper.GetParams(ctx)
+
+	params.FundingFeeMinRate = sdk.NewDecWithPrec(-111, 8)
+	params.FundingFeeMaxRate = sdk.NewDecWithPrec(111, 8)
+
```

Comment:
do we need to update base rate as well ?

Reply:
yes this is the new range in order to align with the 0.1% / hour range